lib/sift/filter.rb

```diff
@@ -51,10 +52,6 @@ def param
 
     private
 
-    def parameterize(value)
```

Comment:
The handlers take over responsibility for parsing the raw values from the request. This allows the scope handler to use it's types to choose the right parsing strategy.